packages/shopping-cart/src/types.ts

```diff
@@ -525,6 +525,9 @@ export interface ResponseCartProduct {
 	is_included_for_100yearplan: boolean;
 
 	product_variants: ResponseCartProductVariant[];
+
+	product_cost_display: string;
```

Comment:
I just removed these properties; please don't put them back. We're about to remove them from the server. You can do product cost formatting locally using `formatCurrency()` and the integer version of these prices.

Reply:
Thanks for the tip. I've switched to determining the original cost by using `formatCurrency()` with the integer version and removed these added types in https://github.com/Automattic/wp-calypso/pull/86272/commits/349ad405b1ce210532e5d348aba061214f234a1a

Reply:
Thanks!